drivers/filesystems/fastfat/create.c

```diff
@@ -2573,6 +2573,7 @@ Return Value:
         DebugTrace(-1, Dbg, "FatOpenVolume -> Iosb.Status = %08lx\n", Iosb.Status);
```

Comment:
I don't fully understand your `Iosb.Status` -> `Status` changes here.
In addition, now in this debug trace the Iosb.Status looks like it wouldn't be the intended value to be displayed anymore.

Reply:
Clang cl is "picky" with it's SEH and doesn't seem to like using the IoStatusBlock in the __finally. It just crashes.
```
PLEASE submit a bug report to https://github.com/llvm/llvm-project/issues/ and include the crash backtrace, preprocessed source, and associated run script.
Stack dump:
1.      C:\ReactOS\reactos\drivers\filesystems\fastfat\create.c:2585:1: current parser token 'IO_STATUS_BLOCK'
2.      C:\ReactOS\reactos\drivers\filesystems\fastfat\create.c:2292:1: LLVM IR generation of declaration 'FatOpenVolume'
3.      C:\ReactOS\reactos\drivers\filesystems\fastfat\create.c:2292:1: Generating code for declaration 'FatOpenVolume'
Exception Code: 0xC0000005
```